build_daemon/test/daemon_test.dart

```diff
@@ -128,27 +138,37 @@ void main() {
   });
 }
 
-Future<String> getOutput(Process daemon) async {
-  return await daemon.stdout
-      .transform(utf8.decoder)
-      .transform(const LineSplitter())
-      .firstWhere((line) => line == 'RUNNING' || line == 'ALREADY RUNNING');
-}
+Future<String> getOutput(Process daemon) async =>
```

Comment:
https://www.dartlang.org/guides/language/effective-dart/design#avoid-starting-a-method-name-with-get

Also, should it be private?

Reply:
maybe `outputOf`?